web/frontend/src/components/utils/Endpoints.tsx

```diff
@@ -6,14 +6,28 @@ export const ENDPOINT_USER_RIGHTS = '/api/user_rights';
 export const ENDPOINT_ADD_ROLE = '/api/add_role';
 export const ENDPOINT_REMOVE_ROLE = '/api/remove_role';
 
-export const ENDPOINT_EVOTING_CREATE = '/api/evoting/create';
-export const ENDPOINT_EVOTING_RESULT = '/api/evoting/result';
-export const ENDPOINT_EVOTING_CLOSE = '/api/evoting/close';
-export const ENDPOINT_EVOTING_CANCEL = '/api/evoting/cancel';
-export const ENDPOINT_EVOTING_DECRYPT = '/api/evoting/decrypt';
-export const ENDPOINT_EVOTING_SHUFFLE = '/api/evoting/shuffle';
-export const ENDPOINT_EVOTING_CAST_BALLOT = '/api/evoting/cast';
+export const ENDPOINT_EVOTING_CREATE = '/api/evoting/elections';
+
+export const ENDPOINT_EVOTING_ELECTION = (ElectionID = undefined) =>
```

Comment:
I don't see how is it useful to have a default URL with `:ElectionID`.
To me those functions should only build the URLs epxecting a valid `ElectionID`. Additionally, we should simplify the naming and maybe use the [same as the proxy](https://github.com/dedis/d-voting/blob/adds-proxy-signed-requests/proxy/mod.go):

```ts
export const newElection = (electionID: string) => `/api/evoting/elections/${electionID}`;
export const newElectionVote = (electionID: string) => ...
```

then when we use this file we use a named import:

```ts
import * as endpoints from ...;
```